### PR TITLE
summer camp 2019 links

### PR DESCRIPTION
**Pull request by hamidzr**

### CODE REVIEW

**Comment by brollb** [comment]
@hamidzr Is there a reason why this wasn't merged? Should it be?

**Comment by hamidzr** [comment]
> @hamidzr Is there a reason why this wasn't merged? Should it be?

Good question, I don't remember.. : )

File: src/server/index.js [hunk 1]
```diff
@@ -110,6 +110,16 @@ app.get('/csta18', (req,res) => renderView(res, 'csta18.pug'));
 app.get('/cybersecurity', (req,res) => renderView(res, 'roboscape.pug'));
 app.get('/roboscape', (req,res) => res.redirect('/cybersecurity'));
 
+// 2019 camp files
+app.get('/camp2019/netsblox', (req,res) => renderView(res, 'camp2019/netsblox.pug'));
```
Comment (brollb) [inline src/server/index.js]: @hamidzr - where are these files? They don't seem to be included in the PR (I missed this when first merging it :/ )...

Comment (hamidzr) [reply src/server/index.js]: maybe we went with `/roboscaspe` instea and thats why this wasn't finished :thinking:
